web/gds-admin-ui/src/components/Auth/SignWithGoogle.js

```diff
@@ -0,0 +1,32 @@
+
+
+import React from 'react'
+
+function SignWithGoogle({ loginResponse, text, clientId }) {
+
+    React.useEffect(() => {
+        window.onload = function () {
+            if (window.google) {
+                window.google.accounts.id.initialize({
```

Comment:
Note that when I was attempting to use these functions even with the correct Client ID - I was getting 403 errors; so we may need to look into that when we get there.

Reply:
Okay

Reply:
It's related to ClientId configuration 
`m=credential_page_library:41 [GSI_LOGGER]: The given origin is not allowed for the given client ID.`

Reply:
I have; I'll keep playing with it, but let's not let us block us from moving forward for now since the server doesn't have authentication added yet.